### ISSUE TITLE
SchedBase::addTask() is not returning a value

### ISSUE THREAD

**Issue by bperrybap** [Nospampls/SchedTask#3]
The function addTask in SchedBase.cpp is not returning any value:

```
int SchedBase::addTask(SchedBase* pBase) {						// add a new task to the dispatch list
		pBase->taskLink = tasksHead;									// link this task to previous head task
		tasksHead = pBase;												// this task is now at the head
		taskCount++;														// update the task count
}
```

**Comment by bperrybap** [comment]
github glitched with a 404 error when issue 2 was created but it ended up posting this issue twice.
This issue is being closed as total duplicate of issue #2

**Comment by Nospampls** [comment]
Thanks for pointing this out.  I'll make addTask return the new taskCount
in the next release.  Since addTask is not Public it should not affect
users either way I assume.
